tests/get_kv_table_nodeos_tests.cpp

```diff
@@ -796,6 +955,53 @@ BOOST_FIXTURE_TEST_CASE( get_kv_table_nodeos_test, TESTER ) try {
    chk_result(0, 4);
    chk_result(1, 5);
 
+   // test next_key
+   p.reverse = false;
+   p.index_name = "ff"_n;
+   p.index_value = "";
+   p.encode_type = "dec";
+
+   p.lower_bound = "-0.02";
+   p.upper_bound = "-0.01";
+   result = plugin.read_only::get_kv_table_rows(p);
+   BOOST_REQUIRE_EQUAL(0u, result.rows.size());
+   BOOST_REQUIRE_EQUAL(result.more, false);
+
+   p.upper_bound = "";
+   p.limit = 1;
+   result = plugin.read_only::get_kv_table_rows(p);
+   BOOST_REQUIRE_EQUAL(1u, result.rows.size());
+   BOOST_REQUIRE_EQUAL(result.more, true);
+   BOOST_REQUIRE_EQUAL(result.next_key != "", true);
```

Comment:
Can you use real value  instead of   next_key != ""  ?
   ex)  next_key == "some real value"

Reply:
I tried using "some real value" for the following checks. One check failed and the other passed.
BOOST_REQUIRE_EQUAL(result.next_key == "1.01000000000000001", true); - Failed
BOOST_REQUIRE_EQUAL(result.next_key == "1.010000", true); - Passed

Is there a way to control the precision of returned result.next_key, which is decoded from result.next_key_bytes?

Reply:
I think you can just check  a sub string of "1.01"

Reply:
Fixed.